consensus/istanbul/core/core.go

```diff
@@ -304,7 +304,7 @@ func (c *core) catchUpRound(view *istanbul.View) {
 	c.roundChangeSet.Clear(view.Round)
 
 	c.newRoundChangeTimer()
-	logger.Warn("[RC] Catch up round", "new_round", view.Round, "new_seq", view.Sequence, "new_proposer", c.valSet)
+	logger.Warn("[RC] Catch up round", "new_round", view.Round, "new_seq", view.Sequence, "new_proposer", c.valSet.GetProposer())
```

Comment:
```suggestion
	logger.Warn("[RC] Catch up round", "new_round", view.Round, "new_seq", view.Sequence, "new_proposer", c.valSet.GetProposer().String())
```

Reply:
The logger will print the value with `String()` function. `c.valSet.GetProposer()` is logged like "0xD1B1F1d7aB5E89B676B98A2A2886a095f5992ca7"  in the current log.